experimental/dds/tree2/src/feature-libraries/schemaIndexFormat.ts

```diff
@@ -30,29 +30,6 @@ import {
 } from "../core";
 import { brand, fail } from "../util";
 
-const Baz = Type.Recursive((Baz2) =>
```

Comment:
Were these just leftover prototyping code that were checked in by accident or something? Just curious, seems good to delete.

Reply:
I don't know which of us did this, but I assume it was an accident.